MeshLib/Vtk/VtkMappedMeshSource.h

```diff
@@ -67,21 +70,24 @@ class VtkMappedMeshSource final : public vtkUnstructuredGridAlgorithm
     VtkMappedMeshSource(const VtkMappedMeshSource &); // Not implemented.
     void operator=(const VtkMappedMeshSource &);      // Not implemented.
 
-    /// Adds a zero-copy array (MeshLib::VtkMappedPropertyVectorTemplate) as
-    /// either point or cell data to the mesh.
+    /// Adds a zero-copy vtk array wrapper.
     /// \param properties MeshLib::Properties object
-    /// \param prop_name The name of the property vector to be mapped from
-    /// vtk-mesh to ogs-mesh
+    /// \param prop_name The name of the property vector to be mapped
     template <typename T>
     bool addProperty(MeshLib::Properties const& properties,
                      std::string const& prop_name) const
     {
         if (!properties.existsPropertyVector<T>(prop_name))
             return false;
-        auto* const propertyVector = properties.getPropertyVector<T>(prop_name);
+        // TODO: Hack removing const
+        auto* propertyVector = const_cast<MeshLib::PropertyVector<T> *>(
+            properties.getPropertyVector<T>(prop_name));
+        if(!propertyVector)
+            return false;
 
-        vtkNew<VtkMappedPropertyVectorTemplate<T> > dataArray;
-        dataArray->SetPropertyVector(const_cast<MeshLib::PropertyVector<T> &>(*propertyVector));
+        vtkNew<vtkAOSDataArrayTemplate<T> > dataArray;
+        dataArray->SetArray(propertyVector->data(),
+            static_cast<vtkIdType>(propertyVector->size()), 1);
```

Comment:
I've tried to fix the failing ctests; for now I've only a hint.
The failing tests all have a multi-component variable (like displacement) which are not correctly written (with only one component) in the output, so vtkdiff is correct there.
The `addProperty` function is entered with correct arguments and the `properties` do hold a "displacement" array with 2 components. I suspect that the new `dataArray` has default value of 1 for the tuple size.